pkg/kotsadm/main.go

```diff
@@ -466,6 +478,24 @@ func removeKotsadmPostgres(deployOptions types.DeployOptions, clientset *kuberne
 	return nil
 }
 
+func removeMinioXlMigrationScriptsConfigMap(deployOptions types.DeployOptions, clientset *kubernetes.Clientset) error {
```

Comment:
let's not remove the scripts so that we can use them for troubleshooting purposes if something goes wrong later. it's just a configmap, so it's not a big deal to leave it around.